guides/common/modules/proc_restoring-from-incremental-backups.adoc

```diff
@@ -8,8 +8,8 @@ When the restore process completes, all processes are online, and all databases
 
 .Procedure
 . Restore the last full backup using the instructions in xref:Restoring_from_a_Full_Backup_{context}[].
-. Remove the full backup data from {ProjectServer}’s local file system, for example, `/var/` or `/var/tmp/`.
-. Copy the incremental backup data to {ProjectServer}’s local file system, for example, `/var/` or `/var/tmp/`.
+. Remove the full backup data from {ProjectServer}'s local file system, for example, `/var/` or `/var/tmp/`.
+. Copy the incremental backup data to {ProjectServer}'s local file system, for example, `/var/` or `/var/tmp/`.
```

Comment:
```suggestion
. Remove the full backup data from the local file system of {ProjectServer}, for example, `/var/` or `/var/tmp/`.
. Copy the incremental backup data to the local file system on {ProjectServer}, for example, `/var/` or `/var/tmp/`.
```

Reply:
I have reworded both lines to "_on_ Foreman Server".